README.md

```diff
@@ -4,9 +4,21 @@
 and systems administration tasks. It allows the user to run commands across a
 server farm.
 
-## Local usage
+## Local usage and environment setup
+
+The tool is intended to be configured and run on your local workstation/laptop. You should use `virtualenv` to create an isolated Python environment.
```

Comment:
This is a safe option for people who want to get up and running with minimal fuss (or indeed python knowledge). As ever, if you're happy doing something else then feel free to do so.